src/pages/basic-concepts/styling.md

```diff
@@ -1,4 +1,510 @@
 ---
 layout: ~/layouts/Main.astro
-title: Styling
----
\ No newline at end of file
+title: 💅Styling
+---
+
+## 🖍 Quick Start
+
+Styling in an Astro component is done by adding a `<style>` tag anywhere. By default, all styles are **scoped**, meaning they only apply to the current component. To create global styles, add a `:global()` wrapper around a selector (the same as if you were using [CSS Modules][css-modules]).
+
+```html
+<!-- src/components/MyComponent.astro -->
+
+<style>
+  /* Scoped class selector within the component */
+  .scoped {
+    font-weight: bold;
+  }
+
+  /* Scoped element selector within the component */
+  h1 {
+    color: red;
+  }
+
+  /* Global style */
+  :global(h1) {
+    font-size: 32px;
+  }
+</style>
+
+<div class="scoped">I’m a scoped style and only apply to this component</div>
+<h1>I have both scoped and global styles</h1>
+```
+
+**Tips**
+
+- `<style>` tags within `.astro` files will be extracted and optimized for you on build. So you can write CSS without worrying too much about delivery.
+- For best result, only have one `<style>` tag per-Astro component. This isn’t necessarily a limitation, but it may result in better optimization at buildtime.
+- If you want to import third-party libraries into an Astro component, you can use [Sass][sass]! In particular, [@use][sass-use] may come in handy (e.g. `@use "bootstrap/scss/bootstrap"`);
+
+## Cross-Browser Compatibility
+
+We also automatically add browser prefixes using [Autoprefixer][autoprefixer]. By default, Astro loads the [Browserslist defaults][browserslist-defaults], but you may also specify your own by placing a [Browserslist][browserslist] file in your project root.
+
+---
+
+## Supported Styling Options
```

Comment:
Yeah this does flow better—addressing the core styling first, then frontend frameworks later.